.github/workflows/ci.yml

```diff
@@ -34,4 +34,8 @@ jobs:
 
       - name: Run lint check
         run: |
-          black --check --verbose deepinfra
\ No newline at end of file
+          black --check --verbose deepinfra
+
+      - name: Run type check
```

Comment:
I'd put the mypy check first. If both have issues I'd rather the pipeline stops because of mypy first.

Reply:
@ichernev  I'll do that. I'm curious with the reasoning behind.